tsconfig.json

```diff
@@ -32,7 +32,11 @@
         "types/*"
       ]
     },
-    "plugins": [{ "name": "typescript-plugin-css-modules" }]
+    "plugins": [
+      {
+        "name": "typescript-plugin-css-modules"
+      }
+    ]
```

Comment:
This probably could be removed (css-modules supported by webpack's `css-loader`)

Reply:
I believe this is here to support intellisense for css modules.

Reply:
Nope, it doesn't. For `VScode` you still have to install some plugin.
Recommended extension: https://marketplace.visualstudio.com/items?itemName=phoenisx.cssvar

Reply:
According to https://www.npmjs.com/package/typescript-plugin-css-modules#visual-studio-code it seems that this is enough.